### PR TITLE
In 136397453 add race details

### PR DESCRIPTION
**Pull request by noraOSI**

### CODE REVIEW

File: app/assets/javascripts/components/people/RacesEditView.jsx [hunk 1]
```diff
@@ -13,27 +13,28 @@ export class RacesEditView extends React.Component {
     if (isChecked) {
       let newRaces
       if (RACES[race].exclusive) {
-        newRaces = Immutable.List([race])
+        newRaces = Immutable.List([{race: race}])
```
Comment (pemaOSI) [inline app/assets/javascripts/components/people/RacesEditView.jsx]: Immutable.List([{race: race}]) could be Immutable.List([{race: SelectedRace/RaceValue}])

Comment (noraOSI) [inline app/assets/javascripts/components/people/RacesEditView.jsx]: ok!

File: app/assets/javascripts/components/people/RacesEditView.jsx [hunk 2]
```diff
@@ -23,28 +24,75 @@ export class RacesEditView extends React.Component {
...
 
-  render() {
+  changeRaceDetail(race, raceDetail) {
     const {races} = this.props
-    const selectedRaces = races.toJS().map(({race}) => race)
-    const exclusiveRaceSelected = selectedRaces.some((race) => RACES[race].exclusive)
+    const index = races.toJS().findIndex((item) => item.race === race)
+    let newRaces
+    if (raceDetail) {
+      newRaces = races.set(index, {race: race, race_detail: raceDetail})
```
Comment (pemaOSI) [inline app/assets/javascripts/components/people/RacesEditView.jsx]: here race: race does it mean selectedRace or raceValue?

Comment (noraOSI) [inline app/assets/javascripts/components/people/RacesEditView.jsx]: yes, I'll change the variable name for clarity

**Pull request merged**